Makefile

```diff
@@ -6,4 +6,12 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
+	docker run --rm \
+		--env CIRCLECI \
+		--env CIRCLE_BUILD_NUM \
+		--env CIRCLE_BRANCH \
+		--env CIRCLE_SHA1 \
+		--env CODECLIMATE_REPO_TOKEN \
+		--workdir /usr/src/app \
+		--volume "$(CURDIR)/.git:/usr/src/app/.git" \
```

Comment:
💄 Pretty sure we normally use `$(PWD)` over `CURDIR`: `PWD` translates better from shell scripts, while `CURDIR` is make-specific.